test/endpoints/baz/baz_simpleservice_method_compare_test.go

```diff
@@ -105,3 +106,110 @@ func BenchmarkCompare(b *testing.B) {
 	b.StopTimer()
 	gateway.Close()
 }
+
+func TestCompare(t *testing.T) {
```

Comment:
The downside is when changes are needed, we have more places to do that. But yeah, it's trivial.